kcidb/templates/fluster_test.j2

```diff
@@ -40,11 +40,14 @@
             {% set statuses_counts = titles_statuses_counts.get(
                                         title, {"PASS": 0, "FAIL": 0}
                                      ) %}
+            {% set fluster_test_count_len = 5 %}
             {{- indent + " " + title +
                 ":" + " " * (stats_indent|length - indent|length - title|length) +
                 "✅ " + statuses_counts.get("PASS", 0) | string +
-                "  ❌ " + statuses_counts.get("FAIL", 0) | string+
-                "  Σ " + statuses_counts.values() | sum | string +
+                " " * fluster_test_count_len +
+                "❌ " + statuses_counts.get("FAIL", 0) | string+
+                " " * fluster_test_count_len +
```

Comment:
This wouldn't really solve the problem, as you're always adding the same number of spaces. They will still be unaligned, just further apart.